### PR TITLE
Implement namespace's use command to save context and platfrom values…

### PR DESCRIPTION
**Pull request by mahkoCosmo**
… in local

### CODE REVIEW

**Review by Leopold-Cramer — APPROVED** [review]

File: Babylon/utils/environment.py [hunk 1]
```diff
@@ -407,3 +407,21 @@ def get_state_id(self):
...
+            ns_dir.mkdir(parents=True, exist_ok=True)
+        s = ns_dir / "namespace.yaml"
+        ns = dict(context=self.context_id, platform=self.environ_id)
+        s.write_bytes(data=yaml.dump(ns).encode("utf-8"))
+
+    def get_namespace_from_local(self):
+        ns_dir = Path().home() / ".config/cosmotech/babylon"
+        ns_file = ns_dir / "namespace.yaml"
+        if not ns_file.exists():
+            return None
```
Comment (vysahjk) [inline Babylon/utils/environment.py:423]: error log is missing.
What if i run get_all command before `babylon namespace use -c test -p dev` ?

File: Babylon/utils/environment.py [hunk 2]
```diff
@@ -407,3 +407,21 @@ def get_state_id(self):
...
+        ns_data = yaml.load(ns_file.open("r"), Loader=yaml.SafeLoader)
+        self.context_id = ns_data.get("context")
+        self.environ_id = ns_data.get("platform")
+        return
```
Comment (vysahjk) [inline Babylon/utils/environment.py:427]: return value is missing

File: Babylon/commands/namespace/use.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+logger = getLogger("Babylon")
+env = Environment()
+
+
+@command()
+@wrapcontext()
+@timing_decorator
+def use() -> CommandResponse:
+    env.store_namespace_in_local()
+    logger.info("Namespace set")
```
Comment (vysahjk) [inline Babylon/commands/namespace/use.py:17]: 1. return value is missing.
2. add logs with context and platform saved

Example:
`kubectl config use-context phoenixAKSdev
> Switched to context "phoenixAKSdev".`

**Review by vysahjk — CHANGES_REQUESTED** [review]

**Review by vysahjk — APPROVED** [review]
👍🏽

**Pull request merged**